README.md

```diff
@@ -2,6 +2,16 @@
 
 SirMordred is the tool used to coordinate the execution of the GrimoireLab platform, via two main configuration files, the `setup.cfg` and `projects.json`, which are summarized below,
```

Comment:
I think this line could be changed as now content lies right below. WDYT?

Reply:
Nice catch!